medallion/backends/mongodb_backend.py

```diff
@@ -209,28 +229,31 @@ def _update_manifest(self, api_root, collection_id, media_type):
     def server_discovery(self):
         discovery_db = self.client["discovery_database"]
         discovery_info = discovery_db["discovery_information"]
-        pipeline = [
-            {
-                "$lookup": {
-                    "from": "api_root_info",
-                    "localField": "api_roots",
-                    "foreignField": "_name",
-                    "as": "_roots",
-                },
-            },
-            {
-                "$addFields": {
-                    "api_roots": "$_roots._url",
-                },
-            },
-            {
-                "$project": {
-                    "_roots": 0,
-                    "_id": 0,
-                }
-            }
-        ]
-        info = discovery_info.aggregate(pipeline).next()
+        # pipeline = [
+        #     {
+        #         "$lookup": {
+        #             "from": "api_root_info",
+        #             "localField": "api_roots",
+        #             "foreignField": "_name",
+        #             "as": "_roots",
+        #         },
+        #     },
+        #     {
+        #         "$addFields": {
+        #             "api_roots": "$_roots._url",
+        #         },
+        #     },
+        #     {
+        #         "$project": {
+        #             "_roots": 0,
+        #             "_id": 0,
+        #         }
+        #     }
+        # ]
+        # info = discovery_info.aggregate(pipeline).next()
+        # return info
```

Comment:
delete

Reply:
This was much more complicated that I think it needs to be, since it is just returning the one record when you use the discovery endpoint.  I wasn't sure that I wasn't missing something, which is why I kept it around, but  I guess that is why we have a source control system